### ISSUE TITLE
UKMM tries to change bootup packs that don't exist

### ISSUE THREAD

**Issue by AftDawn** [NiceneNerd/ukmm#148]
Hi, not entirely sure if i should be posting this issue here or on the GB page of the mod that is causing is, but i feel it's more on UKMM then on the mod.

UKMM is trying to make changes to different language bootup packs that i don't have, i have the `Wii U` `US EN` release of BotW but the mod [Ultimate Xbox UI](https://gamebanana.com/mods/373804) has every type of bootup pack
```
[2023-07-03 12:03:34] INFO Started ukmm
[2023-07-03 12:03:35] INFO Initializing mod manager
[2023-07-03 12:03:35] INFO Current profile: Default
[2023-07-03 12:03:35] INFO Initializing deployment manager
[2023-07-03 12:03:35] INFO No files pending deployment
[2023-07-03 12:03:35] INFO Logger initialized
[2023-07-03 12:03:47] INFO Applying pending changes to mod configuration
[2023-07-03 12:03:47] INFO Applying changes
[2023-07-03 12:03:47] INFO Manifest not provided, remerging all mods
[2023-07-03 12:03:47] INFO Applying changes
[2023-07-03 12:03:47] INFO Unpacking game texts
[2023-07-03 12:03:51] ERROR Failed to apply pending mod changes

Caused by:
    No copy of Pack/Bootup_EUde.pack found in game dump or any mod. Sometimes this is a problem with the mod or your game dump. However, it can also be a temporary cache error. If you know your dump and its configuration are sound, try restarting UKMM. If it persists, copy the error details and post as a comment to https://github.com/NiceneNerd/ukmm/issues/120. at `crates/uk-mod/src/unpack.rs@720:14`
```

Maybe have some way of telling UKMM that the same change is in all of the packs, so only apply what it can and don't worry about the rest?

Unless this is already implemented and it's the mod author that has screwed up, i don't know.

**Comment by neocpu** [comment]
can confirm this does happens with some mods

**Comment by a-plastic-bag** [comment]
Yeah, ran into this problem with some mods that change dialogue too

**Comment by NiceneNerd** [comment]
Now this is solved for real as of [b2cf2a7](https://github.com/NiceneNerd/UKMM/commit/b2cf2a7b76e3f3b0176b3acbbef75a97ebf16ead)
